src/components/Map.js

```diff
@@ -36,17 +37,21 @@ const Map = ({
       const panLat = panCoords.lat();
       const panLng = panCoords.lng();
       setUserHasPanned(true);
-      handleCenterChanged(panLat, panLng);
+      setTimeout(() => {
```

Comment:
So this should work well for the basic case, but let's think through a scenario:

1. Imma User has her delay set to 30 seconds
2. She pans the map
3. 20 seconds later, she pans the map again

What I _think_ would happen is Imma will be waiting most of a minute to see new pins. Imagine she continues to pan, what will happen is we will continue to queue up calls which will then fire 30 seconds after each call to this function.

`setTimeout` returns an ID you can use to `cancelTimeout` so consider stashing that in local state and canceling before you make a new `setTimeout` call.